docs/getting-started/quickstart.md

```diff
@@ -48,10 +48,9 @@ See the [install guide](/getting-started/installation/) for more detailed instal
 
 ## Step 2: Connect to Prefect's API
 
-Much of Prefect's functionality is backed by an API.
-Sign up for a forever-free [Prefect Cloud account](/cloud/) or accept your organization's invite to join their Prefect Cloud account.
+Much of Prefect's functionality is backed by an API. The easiest way to get started is to use the API hosted by Prefect:
 
-1. Create a new account or sign in at [https://app.prefect.cloud/](https://app.prefect.cloud/).
+1. Create a forever-free Prefect Cloud account or sign in at [https://app.prefect.cloud/](https://app.prefect.cloud/).
 1. Use the `prefect cloud login` CLI command to [log in to Prefect Cloud](/cloud/users/api-keys) from your environment.
```

Comment:
```suggestion
1. Create a forever-free Prefect Cloud account or sign in at [https://app.prefect.cloud/](https://app.prefect.cloud/).
2. Use the `prefect cloud login` CLI command to [log in to Prefect Cloud](/cloud/users/api-keys) from your environment.
```

Reply:
Jeff set up this nifty linter that automatically numbers lists correctly